packages/components/transfer/src/transfer.vue

```diff
@@ -145,7 +145,12 @@ watch(
 const optionRender = computed(() => (option: TransferDataItem) => {
   if (props.renderContent) return props.renderContent(h, option)
 
-  if (slots.default) return slots.default({ option })
+  const defaultSlotVNodes = slots.default?.({ option }) || []
```

Comment:
There will still be problems if there are multiple comments.
```suggestion
  const defaultSlotVNodes = (slots.default?.({ option }) || []).filter(node => node.type !== Comment)
  if (defaultSlotVNodes.length) {
    return defaultSlotVNodes 
  }
```

Reply:
Thank you for your advice. I update codes in a new commit and add tests. ♥️